tox.ini

```diff
@@ -1,20 +1,18 @@
 [tox]
-envlist = py27, py27-configparser, py35, py36, py37, py38, pypy, pypy3
+envlist = py35, py36, py37, py38, py38-configparser, pypy3
```

Comment:
the original reason to install `configparser` instead of using the default library's was that the py3 configparser was different (and preferred). Now that we no longer need Python2 we also do not need the third-party configparser, let's remove those options

Reply:
How does https://github.com/c4urself/bump2version/pull/108/commits/ff9fec876b9fb1b69fc7d2fa790d8fa2bf303467 look?